### PR TITLE
Merge 1.7.6.x into 1.7.7.x - 03/03/2020

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Merge last changes from 1.7.6.x into 1.7.7.x
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17947)

### CODE REVIEW

**Comment by matks** [comment]
@PierreRambaud Maybe we need to rebuild assets ?

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
@matks Assets has already been rebuilt :)

File: classes/PaymentModule.php [hunk 1]
```diff
@@ -1108,7 +1108,7 @@ protected function createOrderCartRules(
             // THEN
             //  The voucher is cloned with a new value corresponding to the remainder
             $remainingValue = $cartRule->reduction_amount - $values[$cartRule->reduction_tax ? 'tax_incl' : 'tax_excl'];
-            $remainingValue = Tools::ps_round($remainingValue, $computingPrecision);
+            $remainingValue = Tools::ps_round($remainingValue, _PS_PRICE_COMPUTE_PRECISION_);
```
Comment (jolelievre) [inline classes/PaymentModule.php:1111]: Nope not in 177 where computing precision depends on currency precision

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: classes/PaymentModule.php (hunk 1 above)
Comment (PierreRambaud) [reply classes/PaymentModule.php:1111]: But it's a new change in the 1.7.6.x :thinking:

**Comment by PierreRambaud** [comment]
Closed in favor of #18019 18019

**Pull request closed without merging**